subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -361,33 +448,37 @@ struct bt_mesh_prov bt_mesh_shell_prov = {
 	.static_val = NULL,
 	.static_val_len = 0,
 	.output_size = 6,
-	.output_actions = (BT_MESH_DISPLAY_NUMBER | BT_MESH_DISPLAY_STRING),
+	.output_actions = (BT_MESH_BLINK | BT_MESH_BEEP | BT_MESH_VIBRATE | BT_MESH_DISPLAY_NUMBER |
+			   BT_MESH_DISPLAY_STRING),
 	.output_number = output_number,
 	.output_string = output_string,
 	.input_size = 6,
-	.input_actions = (BT_MESH_ENTER_NUMBER | BT_MESH_ENTER_STRING),
+	.input_actions =
+		(BT_MESH_ENTER_NUMBER | BT_MESH_ENTER_STRING | BT_MESH_TWIST | BT_MESH_PUSH),
 	.input = input,
 	.input_complete = prov_input_complete,
 };
 
 static int cmd_static_oob(const struct shell *sh, size_t argc, char *argv[])
 {
+	struct bt_mesh_prov *prov = (struct bt_mesh_prov *)bt_mesh_prov_get();
```

Comment:
It is not clear to me if `CONFIG_BT_MESH_SHELL_PROV_CTX_INSTANCE` is set to `y` how static OOB value will be set there? 
Could you clarify that?

Reply:
As I understand it the prov context pointer is passed from the app to `bt_mesh_init` which is then passed to  `bt_mesh_prov_init`. prov.c stores this pointer reference and when we call `bt_mesh_prov_get` in shell.c we get access to this pointer. The `cmd_static_oob` will be agnostic to the fact if we are using the internal prov context or not. We could add a "shorcut" for the instance where `CONFIG_BT_MESH_SHELL_PROV_CTX_INSTANCE ` is enabled so that we could use direct access to the internal context, but I don't see the point in doing that if we nevertheless want to access a prov context other than the internal context